src/app/components/AvailableModulesMenu.jsx

```diff
@@ -213,16 +213,30 @@ export default class AvailableModulesMenu extends TranslatedComponent {
               if (categories.length === 1) {
                 // Show category header instead of group header
                 if (m && grp == m.grp) {
-                  list.push(<div ref={(elem) => this.groupElem = elem} key={category}
+                  // If this is a missing module/weapon, skip it
+                  if (m.grp == "mh" || m.grp == "mm"){
+                    continue;
```

Comment:
The reason for the continue, is because there are other things done in the loop, which don't need to be done, as we're deliberately avoiding adding a header for a 'Missing Hardpoint' or 'Missing Module' section, we don't want anything to happen at all for these modules, we don't want the header, or the modules to be added, we don't want counting vars to be incremented, etc. so we continue to the next iteration of the loop. This only satisfies the requirement for a module group that only has one module, further down, there is another continue on line 234, for module groups where there are more than 1.